src/controller/python/ChipDeviceController-ScriptBinding.cpp

```diff
@@ -210,6 +211,13 @@ ChipError::StorageType pychip_DeviceController_NewDeviceController(chip::Control
     initParams.controllerNOC                  = nocSpan;
 
     (*outDevCtrl)->SetUdpListenPort(CHIP_PORT + 1);
+#if CHIP_DEVICE_CONFIG_ENABLE_COMMISSIONER_DISCOVERY // make this commissioner discoverable
+    auto advertiseParameters = chip::Mdns::CommissionAdvertisingParameters();
+    chip::app::Mdns::BuildCommissionAdvertisingParameters(false, chip::app::Mdns::CommissioningMode::kDisabled,
+                                                          advertiseParameters);
+    (*outDevCtrl)->SetCommissionAdvertisingParameters(&advertiseParameters);
```

Comment:
If this is meant to be used only during the initialization, move it to `CommissionerInitParams`. Otherwise, it's unsafe to store a pointer to a local variable.